src/pages/business/business-page.styles.tsx

```diff
@@ -0,0 +1,78 @@
+import { Card, SubHeading, Button, Image } from '@sumup/circuit-ui';
```

Comment:
Again this is hack week, so no problem, but just FYI for "real" work stuff in the future: 

https://github.com/sumup/ze-dashboard/pull/4530#discussion_r675393786

Reply:
Thank you so much for this insight. I didn't know about this.
Very useful piece of information!!!! ❤️ 💪

Reply:
"Hack week is for learning" – @jmaslate

Reply:
😄